src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -902,7 +894,7 @@ public void removeCitizen(CitizenData citizen)
         //  Remove the Citizen
         citizens.remove(citizen.getId());
 
-        for (Building building : buildings.values())
+        for (AbstractBuilding building : buildings.values())
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'building' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)